internal/pkg/archer/project.go

```diff
@@ -7,7 +7,15 @@ package archer
 type Project struct {
 	Name      string `json:"name"`    // Name of a project. Must be unique amongst other projects in the same account
 	AccountID string `json:"account"` // AccountID this project is mastered in
-	Version   string `json:"version"` // The version of the project layout in the underyling datastore (e.g. SSM)
+	Version   string `json:"version"` // The version of the project layout in the underlying datastore (e.g. SSM)
+}
+
+// ProjectRegionalResources represent project resources that are regional
+type ProjectRegionalResources struct {
+	Region         string            // The region these resources are in
+	KMSKeyARN      string            // A KMS Key ARN for encrypting Pipeline artifacts
+	S3Bucket       string            // S3 bucket for Pipeline artifacts
+	RepositoryURLs map[string]string // The image repository URLs by app name
```

Comment:
Question for my own edification: are we only ever going to support ECR repositories?

Reply:
That's right. But I figured I'd keep that detail out of there as much as I can.